eslintrc.json

```diff
@@ -2,6 +2,11 @@
   "extends": [
     "prettier/react"
   ],
+
+  "plugins": [
+    "react"
```

Comment:
We need to explicitly reference the `react` plugin since we're referencing rules from it below.